model/vcs/registry.yaml

```diff
@@ -63,11 +63,41 @@ groups:
             "123",
             "HEAD",
           ]
-      - id: vcs.repository.change.title
+      - id: vcs.revision_delta.type
```

Comment:
a naming suggestions to consider `vcs.revision_delta.direction`

Reply:
Renamed in 90b4625c48bea7c152626db0afd384a2ce44e7b3